site/cds_rdm/utils.py

```diff
@@ -284,3 +286,21 @@ def merge(self, name_source, name_dest):
             )
             # Update the name_dest value
             self.service.update(system_identity, name_dest.get("id"), name_dest)
+
+
+def get_clc_sync_entry(record):
+    """Get the CLC sync entry for the record.
+
+    :param record: The record to get the CLC sync entry for.
+    :return: The CLC sync entry.
+    """
+    try:
+        clc_sync_entry = current_clc_sync_service.read(
+            system_identity,
```

Comment:
I would rather pass the current user here, and then change the `read` permission to allow anyone to read this. Otherwise, we are basically by-passing the read permissions.
WDYT?